doc/config.rst

```diff
@@ -1633,6 +1633,17 @@ These options influence LaTeX output.
         .. versionadded:: 1.2
      ``'fontenc'``
         "fontenc" package inclusion, default ``'\\usepackage[T1]{fontenc}'``.
+     ``'amsmath'``
```

Comment:
ok, should I revert this last commit so that this PR does not add documentation for `'amsmath'` key ?